Dockerfile

```
@@ -0,0 +1,30 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+# install kubectl
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && \
+	chmod +x ./kubectl && \
+	mv ./kubectl /usr/local/bin/kubectl
+
+# install gcloud
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+	apt-get install -y apt-transport-https ca-certificates gnupg && \
+	curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
+	apt-get update && apt-get install -y google-cloud-sdk
+
+COPY . .
+
+#set authentication
+COPY entrypoint.sh /usr/local/bin/
+RUN ln -s /usr/local/bin/entrypoint.sh / # backwards compat
+RUN export GOOGLE_APPLICATION_CREDENTIALS="/usr/src/app/gcloud_key.json"
+
+ENTRYPOINT ["/entrypoint.sh"]
+
+CMD ["python", "secret-script.py", "--get", "--secret_id=docker-secret"]
```

I'm not sure what the intent is here.  I would suggest using ENTRYPOINT or CMD but not both.

---

After reading through this PR I would suggest
```suggestion
CMD ["/entrypoint.sh"]
```